public/data/openapi.yaml

```diff
@@ -185,7 +185,7 @@ components:
     countriesParam:
       name: countries
       in: query
-      description: Liste des codes pays pour chacune des 5 étapes du cycle de vie du produit
+      description: Liste des codes pays pour chacune des 5 étapes du cycle de vie du produit (/!\ ATTENTION, pour le moment, le premier pays n'est pas pris en compte car c'est le pays par défaut de la matière première qui est utilisé pour le calcul)
```

Comment:
I wonder if rapidoc supports Markdown here, so we could add a link to relevant docs?

Reply:
I added a link to https://fabrique-numerique.gitbook.io/wikicarbone/methodologie/filature#liste-complete